src/bitmessageqt/__init__.py

```diff
@@ -18,6 +16,9 @@
 import time
 from datetime import datetime, timedelta
```

Comment:
@coffeedogs can you write import statements first and then go with from statements.